requirements/requirements.txt

```diff
@@ -19,6 +19,9 @@ transformers
 termcolor
 xmlrunner
 pytoda @ git+https://git@github.com/PaccMann/paccmann_datasets/@0.2.5#egg=pytoda
-openfold @ git+https://github.com/aqlaboratory/openfold
+openfold @ git+https://github.com/BiomedSciAI/openfold  
```

Comment:
Did we move it to yoels space or not?

Reply:
@YoelShoshan we talked with Efrat that we move this forked repository under your user, to avoid confusion between original and forked repositories. Can you make this transition?